rpcs3/rpcs3qt/shortcut_utils.cpp

```diff
@@ -246,7 +238,7 @@ namespace gui::utils
 		const std::string launcher_path = macos_dir + "launcher";
 
 		std::string launcher_content;
-		fmt::append(launcher_content, "#!/bin/bash\nopen \"%s\" --args %s", app_bundle_path, target_cli_args);
+		fmt::append(launcher_content, "#!/bin/bash\nopen -b net.rpcs3.rpcs3 --args %s", target_cli_args);
```

Comment:
isn't that different from the CFBundleIdentifier ?

Reply:
It is better to use `[[NSBundle mainBundle] bundleIdentifier]` instead of hardcoded value, isn't it?